cookbooks/ambari/attributes/views.rb

```diff
@@ -0,0 +1,49 @@
+# ambari Views Attributes
```

Comment:
Why is this all commented?

Reply:
This was to refactor code to use `http_request` resource. I guess, I'll delete the file and will get it from git history if needed.